src/Cake.Common/Tools/WiX/WiXAliases.cs

```diff
@@ -4,6 +4,7 @@
 using System;
```

Comment:
@RLittlesII can you add the new license file header? You will find an example on all the other files in the repo. This is a new requirement after joining the .Net Foundation.